crux-core/src/crux/tx.clj

```diff
@@ -383,7 +383,15 @@
 
       (db/put-objects object-store docs)
 
-      (bus/send bus {::bus/event-type ::indexed-docs, :doc-ids (set (keys docs))})
+      (bus/send bus {::bus/event-type ::indexed-docs,
+                     :doc-ids (set (keys docs))
+                     :av-count (->> (vals docs)
```

Comment:
we don't need to call `set` here, and I reckon it might be simpler to combine the AVs first and then count?